libs/stache/src/lib/modules/router/route.service.spec.ts

```diff
@@ -270,11 +270,11 @@ describe('StacheRouteService', () => {
 
   it('should not unset the active routes on NavigationEnd', async () => {
     await setupTest();
-    (router as any).events = observableOf(new NavigationEnd(0, '', ''));
     spyOn(StacheRouteService.prototype, 'clearActiveRoutes');
+    await router.navigate(['a']);
     expect(
       StacheRouteService.prototype.clearActiveRoutes
-    ).not.toHaveBeenCalled();
+    ).toHaveBeenCalledTimes(1);
```

Comment:
Angular changed `router.events` to be read-only.

Reply:
👍